helper/identity/mfa/types.proto

```diff
@@ -85,6 +85,8 @@ message OktaConfig {
   string base_url = 4;
   // @inject_tag: sentinel:"-"
   bool primary_email = 5;
+  // @inject_tag: sentinel:"-"
+  bool use_passcode = 6;
```

Comment:
This mirrors the equivalent property in `DuoConfig`:
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/helper/identity/mfa/types.proto#L71-L72